src/devices/Vl53L0X/Info.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Vl53L0X
```

Comment:
`Iot.Device.Vl53l0x` 😄

Reply:
why not capital L and capital X? Asking for coherence with the rest

Reply:
@ZhangGaoxing there is very high likelyhood we will rename all projects with `x` to `X` so I'd leave as is (reason is because in case of `Bmx280` it looks like part of the device name and not like a placeholder)